gatsby/src/components/index/NewsAndEvents.js

```diff
@@ -123,7 +123,7 @@ const NewsAndEvents = ({ data, language }) => {
               <div className="grid-col">
                 <h2 className="font-heading-xl margin-top-0 tablet:margin-bottom-0 text-center">
                   <div>
-                    <Icon path={mdiNewspaper} title="News Story" size={2.5} />
+                    <Icon path={mdiNewspaper} title={newsTitle} size={2.5} />
```

Comment:
I thought maybe this should be consistent w/ the Events icon, where the icon title matches the header text.

Reply:
Also, just like with the Events icon, I noticed that ChromeVox actually reads `{newsTitle}` twice: once here and on line 128.

Reply:
👍